src/pages/Notes.jsx

```diff
@@ -41,19 +42,45 @@ const {notesState:{notesList},setNotes,noteInput}=useNotes();
                             </select>
                         </div>
                     </div>
+                </div> */}
+                
+                <div class="flex flex-wrap justify-around mt-8 items-center gap-6">
+                <div class="flex">
+                        <button type="button" class="text-white bg-cyan-700 hover:bg-cyan-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center inline-flex items-center mr-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800 relative" onClick={(e)=>{
+                            tooglenotesModal(e);
+                            setNotes(noteInput)
+                            }}>
+                           Add Notes
+                            <IoMdAdd size={22} className="ml-2"/>
+                        </button>
+                    </div>
+                    
+                    <div class="flex pr-6">
```

Comment:
I see few class instead of className. You can change that

Reply:
fixed .Thanks for pointing out.